Assets/Mirror/Components/Experimental/NetworkTransformBase.cs

```diff
@@ -117,28 +113,25 @@ void FixedUpdate()
             // if server then always sync to others.
             // let the clients know that this has moved
             if (isServer && HasEitherMovedRotatedScaled())
-            {
                 ServerUpdate();
-            }
 
             if (isClient)
             {
                 // send to server if we have local authority (and aren't the server)
                 // -> only if connectionToServer has been initialized yet too
                 if (IsOwnerWithClientAuthority)
-                {
                     ClientAuthorityUpdate();
-                }
                 else if (goal.isValid)
-                {
                     ClientRemoteUpdate();
-                }
             }
         }
 
         void ServerUpdate()
         {
-            RpcMove(targetTransform.localPosition, targetTransform.localRotation, targetTransform.localScale);
+            if (clientAuthority)
+                RpcMove(targetTransform.localPosition, targetTransform.localRotation, targetTransform.localScale);
```

Comment:
why does server send anything to client if client has authority?

Reply:
That's part of ServerTeleport - copied directly from the current NT.

Reply:
The teleport part should be in its own functions, it shouldn't need to effect the update loop?